python-sdk/src/astro/databases/mssql.py

```diff
@@ -392,6 +392,34 @@ def _build_merge_sql(
         }
         return statement, params
 
+    def openlineage_dataset_name(self, table: BaseTable) -> str:
+        """
+        Returns the open lineage dataset name as per
+        https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md
+        Example: schema_name.table_name
+        """
+        database = self.hook.get_connection(self.conn_id).schema
+        schema = "dbo"
+        if table.metadata and table.metadata.schema:
+            schema = table.metadata.schema
+        return f"{database}.{schema}.{table.name}"
+
+    def openlineage_dataset_namespace(self) -> str:
+        """
+        Returns the open lineage dataset namespace as per
+        https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md
+        Example: postgresql://localhost:5432
```

Comment:
this example is not correct, it should be mysql?

Reply:
Done